### PR TITLE
Updated readme for astronomy pipeline: alex

### LINKED ISSUE #123: Update Astronomy API pipeline README
As a developer, I want clear documentation that outlines the installation process, the structure of the pipeline, and how to run the app, so new contributors can easily get started and understand the functionality of the app.

- Edit README so that it is up to date with astronomy api pipeline progress

### PR DESCRIPTION
**Pull request by AlexOBarnes**
This closes #123

What I have done:
- Updated readme for the astronomy pipeline

### CODE REVIEW

File: astronomy-pipeline/README.md [hunk 1]
```diff
@@ -2,7 +2,97 @@
...
+pip install -r requirements.txt
+```
+4. Create a `.env` file with the following:
+    - DB_HOST - to access your RDS instance
+    - DB_USER - username for accessing RDS
+    - DB_PASSWORD - password for accessing RDS
+    - DB_NAME - the name of your database
+    - DB_PORT - port to access RDS (typically 5432 for postgres)
+    - ASTRONOMY_ID - obtained when signing up for the astronomy API
+    - ASTRONOMY_SECRET - secret key used for accessing the astronomy API
```
[the remark this answers is not in the document]
Comment (olliejth) [reply astronomy-pipeline/README.md:24]: NEVER

File: astronomy-pipeline/README.md [hunk 2]
```diff
@@ -2,7 +2,97 @@
...
+#### `astronomy_pipeline.py`
+- Written to conform to AWS lambda conventions
+- Orchestrates entire ETL pipeline
+#### `test_astronomy_[filename].py`
+- Contains the tests for the files within this folder
+- Uses `pytest` and `unittest.mock` for each test
+#### `deploy.sh`
+- Runs a series of commands that dockerise the pipeline using the local `Dockerfile`.
+- Pushes the docker image to a given ECR repository
+#### `db_connect.sh`
```
Comment (olliejth) [inline astronomy-pipeline/README.md:97]: Good consistent formatting in having these file name headings all next to one another.

File: astronomy-pipeline/README.md [hunk 3]
```diff
@@ -2,7 +2,97 @@
 This folder contains the code for the `Astronomy API` data pipeline. This includes `extract.py`, `transform.py`, `load.py` and `pipeline.py`. This pipeline uses the `requests` library to send HTML GET requests to the [Astronomy API](https://docs.astronomyapi.com/) and the `psycopg2` library to load this data onto our database.
 
 ## Setup
+1. Ensure that an SQL server RDS has been setup prior and is accessible
+    - Note: be sure to store the credentials for accessing this database safely.
+
+2. Signup for the [Astronomy API](https://astronomyapi.com/auth/signup) and store the id and secret key you receive securely
```
Comment (olliejth) [inline astronomy-pipeline/README.md:8]: Nice addition of the API link.

Comment (olliejth) [inline astronomy-pipeline/README.md:8]: Missing full-stops after 1 + 2 bullet points.

**Review by olliejth — CHANGES_REQUESTED** [review]
Check for missing full stops.

File: astronomy-pipeline/README.md (hunk 3 above)
Comment (AlexOBarnes) [reply astronomy-pipeline/README.md:8]:
```suggestion
2. Signup for the [Astronomy API](https://astronomyapi.com/auth/signup) and store the id and secret key you receive securely.
```

File: astronomy-pipeline/README.md [hunk 4]
```diff
@@ -2,7 +2,97 @@
 This folder contains the code for the `Astronomy API` data pipeline. This includes `extract.py`, `transform.py`, `load.py` and `pipeline.py`. This pipeline uses the `requests` library to send HTML GET requests to the [Astronomy API](https://docs.astronomyapi.com/) and the `psycopg2` library to load this data onto our database.
 
 ## Setup
+1. Ensure that an SQL server RDS has been setup prior and is accessible
```
Comment (AlexOBarnes) [inline astronomy-pipeline/README.md:5]:
```suggestion
1. Ensure that an SQL server RDS has been setup prior and is accessible.
```

**Review by olliejth — APPROVED** [review]
Changes made, approved.

**Pull request merged**